adserver/models.py

```diff
@@ -1005,6 +1005,17 @@ def days(self):
             return []
         return self.targeting_parameters.get("days", [])
 
+    @property
+    def niche_targeting(self) -> float:
+        """
+        Return the niche targeting parameter for this flight.
+
+        Testing out return types here as well..
+        """
+        if not self.targeting_parameters:
+            return float(0)
+        return self.targeting_parameters.get("niche_targeting", float(0))
```

Comment:
```suggestion
            return 0.0
        return self.targeting_parameters.get("niche_targeting", 0.0)
```

This is minor but I believe this syntax is preferred to calling the initializer. (eg. use `= []` vs. `= list()`)